bolt/lib/Target/AArch64/AArch64MCPlusBuilder.cpp

```diff
@@ -133,6 +133,35 @@ class AArch64MCPlusBuilder : public MCPlusBuilder {
 public:
   using MCPlusBuilder::MCPlusBuilder;
 
+  MCPhysReg getStackPointer() const override { return AArch64::SP; }
+
+  bool isPush(const MCInst &Inst) const override { return false; }
```

Comment:
here should be functionality for checking isStoreToStack, where you need to check all store instructions